HTML/3. lists.md

```diff
@@ -0,0 +1,38 @@
+# Lists
```

Comment:
If you want, you could toss in `<dl>`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
Ya wait I learned about `<dl>`s in school so I think it should be mentioned...

P.S. How do you suggest a change. When I review I can suggest a change but that doesn't actually let you add the change directly. It just adds a bunch of comments.

Reply:
Well, you seem to know how to suggest a change already?

CTRL+G, or clicking the icon with an underlined plus-minus.

Reply:
```suggestion
DAFSLKJSDFLKASD
```

Reply:
Yeah okay I'm planning on a webpage with tables, and description lists